cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - name: IMAGE
           value: $(params.IMAGE_NAME)
         - name: VERSION
-          value: "8"
+          value: "12"
```

Comment:
why not the same version as in clustertask, i.e. `12-ubi8`? or the newest available `14-ubi8`?